sources/logdog/conf/logdog.common.conf

```diff
@@ -14,3 +14,4 @@ exec settings.json apiclient --method GET --uri /
 exec signpost signpost status
 exec wicked wicked show all
 file os-release /etc/os-release
+glob /var/log/kdump/*.*
```

Comment:
Do we believe there will always be a `.` in the name?  Are we trying to prevent collection of something that doesn't have a `.`?

Reply:
> Do we believe there will always be a . in the name? 
Yes, we name the files in `prairiedog`, 

>   Are we trying to prevent collection of something that doesn't have a .?
No, I think I can use `*`, I'll test.